data/spells/scripts/support/animate_dead_rune.lua

```diff
@@ -1,5 +1,5 @@
 function onCastSpell(creature, variant, isHotkey)
```

Comment:
But this is a rune spell #1020 , it was originally made for rune spells, so I just removed those from instant spells.